.github/temp/intrin.ps1

```diff
@@ -0,0 +1,9 @@
+
+$intrin = 'C:/Program Files (x86)/Microsoft Visual Studio/2019/Enterprise/VC/Tools/MSVC/14.28.29333/include/intrin0.h'
```

Comment:
As an aside, is this version of MSVC fixed, or is it upgraded automatically in GitHub actions?

Reply:
The current version is broken and requires this hack. I'm not sure about it but I do think they get upgraded automatically in GHA.